pkg/rancher-desktop/assets/translations/en-us.yaml

```
@@ -166,7 +166,24 @@ suffix:
 app:
   name: Rancher Desktop
   update: "There's one last step to finish updating Rancher Desktop"
-
+marketplace:
+  title: Marketplace
+  noResults: No extension found for the searching criteria
```

I think "search criteria" will read a little more naturally 🙂

```suggestion
  noResults: No extension found for the search criteria
```